set.mm

```diff
@@ -491313,9 +491717,9 @@ of the indicator function (Contributed by Thierry Arnoux,
       SULZHSULZAEVRZOUVTUWODVUOUVGJVQVUNUWOAUVGEUXRUVGRZVUMUWNHSVUPVULUWMGSVUPV
       UKUWLFSVUPVUJUWKUWHUXRUVGUWJUUJUUKYSYSYSUULUUMUUNUUOUUTUUPUUQUVRUVLBUVDUF
       UVMUVDRZUVNUVFUVQUVKUVMUVDUVDUGYTVUQUVPUVJCEVUQUVOUVHUVIUVMUVDUVGUJYTUURU
-      USXRUVAUS @.
-  @}
-$)
+      USXRUVAUS $.
+  $}
+
 
 $( Begin $[ set-mbox-ta-geom.mm $] $)
```

Comment:
By the way: do you use this sub-mathbox splitting ?

Reply:
I used to, but not anymore, actually.